BloogBot/packages.config

```diff
@@ -3,9 +3,15 @@
   <package id="Discord.Net.Core" version="2.2.0" targetFramework="net461" />
   <package id="Discord.Net.Rest" version="2.2.0" targetFramework="net461" />
   <package id="Discord.Net.WebSocket" version="2.2.0" targetFramework="net461" />
+  <package id="EntityFramework" version="6.4.4" targetFramework="net461" />
```

Comment:
Can you double check all of these dependencies are required? EntityFramework for example is a pretty heavy dependency, and I don't think you're actually using it.